docker/prod/uwsgi.ini

```diff
@@ -1,4 +1,6 @@
 [uwsgi]
+uid = www-data
+gid = www-data
```

Comment:
Not sure about this actually, since we already use `chpst` to run the process as `art:art` above.